web-docs/concepts/index.md

```diff
@@ -0,0 +1,190 @@
+# flagd basics
+
+Your flagd journey will start by defining your feature flags.
+
+flagd will then read those feature flags and make them available to your application.
+
+Your application will interact with flagd via the [OpenFeature SDK](https://example.com) to retrieve flag values via the flagd API.
+
+![flagd architecture](../images/flagd-logical-architecture.jpg)
```

Comment:
minor - do we need the architecture diagram again here?

Reply:
I think so. People may land on this page instead of the homepage.